client/scss/components/_blockquote.scss

```diff
@@ -0,0 +1,54 @@
+.blockquote {
+  @include font('HNM2');
+  text-align: center;
+
+  &--wb {
+    @include font('WB5');
+  }
+
+  &--lr {
+    @include font('LR2');
+    text-align: left;
+  }
+
+  footer {
```

Comment:
Could / should we try to avoid styling elements?
You can quickly land up in overwrite hell.